test/geometry_visual_test.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <mapnik/util/fs.hpp>
+#include <mapnik/projection.hpp>
+#include <mapnik/geometry_transform.hpp>
+#include <mapnik/util/geometry_to_geojson.hpp>
+
+#include "catch.hpp"
+#include "encoding_util.hpp"
+#include "test_utils.hpp"
+#include "vector_tile_strategy.hpp"
+#include "vector_tile_projection.hpp"
+
+TEST_CASE( "geometries", "should reproject, clip, and simplify") {
+
+    std::vector<std::string> geometries = mapnik::util::list_directory("./test/geometry-test-data/input");
+    for (std::string file: geometries)
+    {
+        std::cout << file << std::endl;
+        mapnik::geometry::geometry<double> geom = testing::read_geojson(file);
+        mapnik::projection longlat("+init=epsg:4326",true);
+        mapnik::proj_transform prj_trans(longlat,longlat); // no-op
+        unsigned tile_size = 256;
+        mapnik::vector_tile_impl::spherical_mercator merc_tiler(tile_size);
+        double minx,miny,maxx,maxy;
+        merc_tiler.xyz(9664,20435,15,minx,miny,maxx,maxy);
+        mapnik::box2d<double> z15_extent(minx,miny,maxx,maxy);
+        mapnik::view_transform tr(tile_size,tile_size,z15_extent,0,0);
+        mapnik::vector_tile_impl::vector_tile_strategy_proj vs(prj_trans, tr, 16);
+        mapnik::box2d<double> clip_extent(std::numeric_limits<double>::min(),
+                                       std::numeric_limits<double>::min(),
+                                       std::numeric_limits<double>::max(),
+                                       std::numeric_limits<double>::max());
+        mapnik::vector_tile_impl::transform_visitor<mapnik::vector_tile_impl::vector_tile_strategy_proj> skipping_transformer(vs, clip_extent);
+        mapnik::geometry::geometry<std::int64_t> new_geom = mapnik::util::apply_visitor(skipping_transformer,geom);
+        std::cout << compare(new_geom) << std::endl;
```

Comment:
@artemp @flippmoke @springmeyer right now I'm stuck trying to figure out if I have created `new_geom` correctly. It seems that right now the `new_geom` geometry produced by the `apply_visitor` function is a non-initialized geometry that throws an error when passed to `compare`.

There are most likely plenty of issues with my code as it stands, but I would appreciate some guidance if any of you have the chance. Once I am able to see tiled vector geometries outputted by the above code, I can start seeing what the expected fixtures should look like.